packages/core-ethereum/docs/modules.md

```diff
@@ -0,0 +1,39 @@
+[@hoprnet/hopr-core-ethereum](README.md) / Exports
+
+# @hoprnet/hopr-core-ethereum
+
+## Table of contents
+
+### Modules
+
+- [channel](modules/channel.md)
+- [channel.spec](modules/channel_spec.md)
+- [commitment](modules/commitment.md)
+- [commitment.spec](modules/commitment_spec.md)
+- [constants](modules/constants.md)
+- [contracts](modules/contracts.md)
+- [contracts/HoprChannels](modules/contracts_hoprchannels.md)
+- [contracts/HoprDistributor](modules/contracts_hoprdistributor.md)
+- [contracts/HoprForwarder](modules/contracts_hoprforwarder.md)
+- [contracts/HoprToken](modules/contracts_hoprtoken.md)
+- [contracts/HoprWrapper](modules/contracts_hoprwrapper.md)
+- [contracts/commons](modules/contracts_commons.md)
+- [contracts/factories/HoprChannels\_\_factory](modules/contracts_factories_hoprchannels__factory.md)
+- [contracts/factories/HoprDistributor\_\_factory](modules/contracts_factories_hoprdistributor__factory.md)
+- [contracts/factories/HoprForwarder\_\_factory](modules/contracts_factories_hoprforwarder__factory.md)
+- [contracts/factories/HoprToken\_\_factory](modules/contracts_factories_hoprtoken__factory.md)
+- [contracts/factories/HoprWrapper\_\_factory](modules/contracts_factories_hoprwrapper__factory.md)
+- [ethereum](modules/ethereum.md)
+- [fixtures](modules/fixtures.md)
+- [index](modules/index.md)
+- [indexer](modules/indexer.md)
+- [indexer/fixtures](modules/indexer_fixtures.md)
+- [indexer/index.spec](modules/indexer_index_spec.md)
+- [indexer/types](modules/indexer_types.md)
+- [indexer/utils](modules/indexer_utils.md)
+- [indexer/utils.spec](modules/indexer_utils_spec.md)
+- [nonce-tracker](modules/nonce_tracker.md)
+- [nonce-tracker.spec](modules/nonce_tracker_spec.md)
+- [transaction-manager](modules/transaction_manager.md)
+- [transaction-manager.spec](modules/transaction_manager_spec.md)
```

Comment:
It could make sense to exclude all *.spec files

Reply:
Ok, done.